src/Background.cpp

```diff
@@ -27,7 +27,7 @@ using namespace Graphics;
 
 namespace {
 	static const Uint32 BG_STAR_MAX = 500000;
-	static const Uint32 BG_STAR_MIN = 50000;
+	static const Uint32 BG_STAR_MIN = 1;
```

Comment:
The reason this was 50,000 was so that there are always some stars generated. With it at 1 you could literally have a starfield with just 1 star.

Reply:
maybe there's someone who wants a pitch black sky :)